src/main/java/com/minecolonies/coremod/util/BarbarianUtils.java

```diff
@@ -0,0 +1,207 @@
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.entity.ai.mobs.EntityArcherBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.EntityBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.EntityChiefBarbarian;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityList;
+import net.minecraft.init.Items;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.World;
+
+import java.util.stream.IntStream;
+
+/**
+ * Utils for The Barbarian Raid Event
+ */
+public class BarbarianUtils
+{
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private BarbarianUtils()
+    {
+    }
+
+    /**
+     * ResourceLocations for barbarians
+     */
+    /* default */ public static ResourceLocation barbarian = EntityList.getKey(EntityBarbarian.class);
+    /* default */ public static ResourceLocation archer    = EntityList.getKey(EntityArcherBarbarian.class);
+    /* default */ public static ResourceLocation chief     = EntityList.getKey(EntityChiefBarbarian.class);
+
+    /* default */ private static final int LEVEL_AT_WHICH_TO_NOT_TRIGGER_RAID = 1;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)